gcf/custom/layout.go

```diff
@@ -62,8 +66,10 @@ func BuildLayout(root string, objects []string) (*Layout, error) {
 	}
 	// Put files under each Import
 	for _, parts := range files {
-		if len(parts) < 2 {
-			logging("Ignore file", parts)
+		if len(parts) == 1 {
+			if parts[0] == provFile {
+				layout.prov = provFile
+			}
 			continue
```

Comment:
nit: Can we keep the log please? It would be helpful to know if a file was ignored.

Reply:
Good catch, not meant to remove those.